app/budget/directives/expenseItems/expenseItems.js

```diff
@@ -0,0 +1,90 @@
+import { toCurrency } from 'shared/helpers/string';
+
+import './expenseItems.css';
+
+let expenseItems = function ($rootScope, BudgetActions) {
+  return {
+    restrict: 'E',
+    template: require('./expenseItems.html'),
+    replace: true,
+    scope: {
+      selectedBudgetScenario: '<',
+      ui: '<',
+      expenseItems: '<',
+      // '<' This is proper one way binding, as opposed to string interpoation or passing value as a function that can be called
+    },
+    link: function (scope) {
+      scope.addExpenseItem = function (expenseItem) {
+        BudgetActions.createExpenseItem(
+          expenseItem,
+          scope.selectedBudgetScenario.id,
+          expenseItem.message
+        );
+      };
+
+      scope.hideExpenseItem = function (expenseItem) {
+        expenseItem.hidden = true;
```

Comment:
Does expenseItem have a hidden attribute?

Reply:
Nope, got rid of those thanks.